README.md

```diff
@@ -1,2 +1,75 @@
-# sidekiq-monitoring
-A rails engine to monitor sidekiq internals.
+# sidekiq_monitoring
+
+Rails engine that provides a JSON API which serves Sidekiq
+status information from a HTTP-auth protected endpoint.
+This status information is gathered by scheduling a frequently
+running job that saves timestamps in Redis.
+
+## Installation
+
+Add gem to your project:
+
+    gem 'sidekiq_monitoring'
+
+Mount engine in `routes.rb`:
+
+    Rails.application.routes.draw do
+      mount SidekiqMonitoring::Engine => "/sidekiq_monitoring"
+    end
+
+Schedule job in `whenever`'s `schedule.rb`:
+
+    every 5.minutes do
+      runner 'SidekiqMonitoring::Status.refresh'
+    end
+
+Be sure that your parent app provides a redis connection via `Redis.current`.
+
+Provide Redis URL and HTTP basic authentication credentials via initializer
+
+    # app/config/initializers/sidekiq_monitoring.rb
+    SidekiqMonitoring.redis_url = 'redis://127.0.0.1:6379/0'
```

Comment:
why do we need this when we want `Redis.current` ?

Reply:
See comment below 👇

Reply:
which one?

Reply:
As fas as I understand it, we could not either: 
Have `Redis.current` set by the application. Or provide a redis_url that works, right?

Reply:
Also in theory we could document the parent controller feature. But I think it could also be ok to leave that undocumented.

Reply:
I'll add a comment to stress that the `redis_url` setting is optional